gov_uk_dashboards/components/plotly/side_navbar.py

```diff
@@ -34,7 +34,12 @@ def side_navbar_link_active(text, href):
     already viewing the linked dashboard.
     """
     return html.Li(
-        dcc.Link(text, href=href, className="govuk-link govuk-link--no-visited-state"),
+        html.A(
+            text,
+            href=href,
+            className="govuk-link govuk-link--no-visited-state",
+            **{"aria-current": "true"}
+        ),
         className=(
```

Comment:
I think the html.A causes a page reload whereas dcc.Link doesn't do a full reload - don't know if it matters as would only be if you click on the active page link again but perhaps worth checking with Joe and possibly trying to wrap the dcc.Link in a dash html element to get the smooth reloading and aria attribute at the same time?

Reply:
Thanks Ellie - we attempted this but couldn't get it to work. Would like to get Joe to review and assess whether he's satisfied with the behaviour.